Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo

```diff
@@ -0,0 +1,33 @@
+within Modelica.Electrical.QuasiStatic.SinglePhase.Sensors;
+model CurrentQuasiRMSSensor "Continuous quasi current RMS sensor for single-phase system"
+  extends Modelica.Electrical.QuasiStatic.SinglePhase.Interfaces.RelativeSensorElementary;
+  Modelica.Blocks.Interfaces.RealOutput I(unit="A")
+    "Continuous quasi average RMS of current" annotation(
+    Placement(visible = true, transformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90), iconTransformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  SI.ComplexCurrent i "Complex current";
+  SI.ComplexVoltage v "Complex voltage";
+equation
+  I = sqrt(i.re^2 + i.im^2);
+  i = pin_p.i;
+  v = pin_p.v - pin_n.v;
+  v = Complex(0,0);
+  annotation (Documentation(info="<html><head></head><body><p style=\"font-size: 12px; font-family: 'DejaVu Sans Mono';\">This sensor determines the continuous quasi&nbsp;<a href=\"Modelica://Modelica.Blocks.Math.RootMeanSquare\">RMS</a>&nbsp;value of a single-phase current system.</p><pre style=\"font-size: 12px;\">  I = abs(i)</pre>
```

Comment:
Please delete all the html stuff not being used within MSL: `<head>` and `<body>`, and all the paragraph formating. Only limited html subset is commonly used, check `Modelica.UsersGuide.Conventions.Documentation`.

Reply:
Should be fixed now, thanks.